tools/eslint-rules/rules-utils.js

```diff
@@ -12,6 +12,16 @@ module.exports.isRequired = function(node, modules) {
     modules.includes(node.arguments[0].value);
 };
 
+/** 
+* Return true if common module is required
+* in AST Node under inspection
+*/
+module.exports.isCommonModule = function(node) {
+  var regExp = new RegExp(/^(\.\.\/)*common(\.js)?$/);
```

Comment:
Why do you not write the RegExp directly? There is no dynamic part as far as I see it.
So `/^(\.\.\/)*common(\.js)?$/` does the job and can be moved outside of the function. That way it is only created once.

Reply:
done.

Reply:
done.